modules/op-release-notes-1-7.adoc

```diff
@@ -409,7 +409,17 @@ time="2022-03-04T09:47:57Z" level=error msg="error writing \"0 0 4294967295\\n\"
 time="2022-03-04T09:47:57Z" level=error msg="(unable to determine exit status)"
 ----
 +
-With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah clustertasks. As a result, the S2I build tasks run successfully.
+With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for `Buildah` and `S2I` clustertasks. As a result, the `Buildah` and `S2I` build tasks can run successfully.
```

Comment:
Please replace instances of `clustertask` that are not objects or commands with `cluster task`. Searching the openshift-docs repo confirms this pattern.

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references

Reply:
@gabriel-rh FYI ^^^